datadog/api/__init__.py

```diff
@@ -35,3 +35,7 @@
 from datadog.api.service_checks import ServiceCheck
 from datadog.api.tags import Tag
 from datadog.api.users import User
+from datadog.api.aws_integration import Aws
+from datadog.api.aws_log_integration import AwsLogs
```

Comment:
Shouldn't this be part of the AWSIntegration?

Reply:
I'd prefer to keep `AwsIntegration` and `AwsLogsIntegration` separate if possible, as these encompass pretty different functionality that is owned by two separate teams.

We've also split these into their own sections in a coming docs update for clarity from a user perspective. 

For me, it also just feels less cluttered having them separate as they each have their own respective GET, PUT, POST, DELETE endpoints for different routes.